.github/workflows/postgres_dev.yaml

```diff
@@ -0,0 +1,70 @@
+name: PG dev
+on:
+  schedule:
+    # run daily 20:00 on master branch
+    - cron: '0 20 * * *'
+  push:
+#    branches:
+#      - prerelease_test
+  pull_request:
+jobs:
+  regress:
+    name: PG14
+    runs-on: ubuntu-20.04
+    env:
+      PG_SRC_DIR: pgbuild
+      PG_INSTALL_DIR: postgresql
+      MAKE_JOBS: 2
+      PG_COMMIT: 5028981923
```

Comment:
A description about what to update to handle more commits would be good if we are going to share this burden of handling PostgreSQL changes. This field needs to be updated, but anything else?

Reply:
just PG_COMMIT needs to be updated

Reply:
Could you just add a comment about that so that the next developer does not start to look for other places that needs changing?